test/views/declaration/destinationCountries/OriginationCountryViewSpec.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.declaration.destinationCountries
+
+import controllers.declaration.routes
+import forms.declaration.destinationCountries.DestinationCountries
+import forms.declaration.destinationCountries.DestinationCountries.OriginationCountryPage
+import models.Mode
+import play.api.data.Form
+import unit.tools.Stubs
+import views.declaration.spec.UnitViewSpec
+import views.html.declaration.destinationCountries.origination_country
+
+class OriginationCountryViewSpec extends UnitViewSpec with Stubs {
+
+  val form: Form[String] = DestinationCountries.form(OriginationCountryPage)
+
+  val originationCountryPage = new origination_country(mainTemplate)
+  val view = originationCountryPage(Mode.Normal, form)
```

Comment:
Please test this for each declaration type

Reply:
This view doesn't change and it's not related with any specific declaration type. I'll change behaviour of all of this, to align with specific journeys, this is just starting point for everything. It'll be added in the future